CMakeLists.txt

```diff
@@ -72,11 +72,17 @@ set(CURSES_NEED_NCURSES TRUE)
 # Dependencies
 find_package(FS REQUIRED)
 find_package(Numa)
+find_package(LLVM 5.0.0)
 find_package(Tbb REQUIRED)
 find_package(Readline REQUIRED)
 find_package(Curses REQUIRED)
 find_package(Sqlite3 REQUIRED)
 
+if(${LLVM_FOUND})
+    set(LLVM_LIBRARY ${LLVM_LIBRARY_DIR}/${CMAKE_SHARED_LIBRARY_PREFIX}LLVM${CMAKE_SHARED_LIBRARY_SUFFIX})
```

Comment:
This could use a comment as to why we're not just linking against the obvious LLVM_LIBRARIES